tasks/adventure_tasks.rst

```diff
@@ -0,0 +1,367 @@
+Adventure Tasks
+===============
+
```

Comment:
maybe also a Table of contents would be handy here and also a link to this from the directory readme?